### DISCUSSION TITLE
tutorials/voyage-part2/01-airlock-the-monolith

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#153] [course feedback]
# tutorials/voyage-part2/01-airlock-the-monolith

https://www.apollographql.com/tutorials/voyage-part2/01-airlock-the-monolith

**Comment by deepakpoojary** [comment]
nice

**Comment by stephan-buckmaster** [comment]
I could imagine a mechanism whereby the schema is split up into several files, maybe even one per type. This could also facilitate collaborating, and not be as heavy as designing supergraphs.
